CONTRIBUTORS.txt

```diff
@@ -0,0 +1,6 @@
+Maintainers
```

Comment:
This file is a mess to maintain, I would use the git history and github autorisation as sources of truth. (Pylint has something similar for historical reasons only)

Reply:
True indeed, I will have a look at Pylint and fix this.

Reply:
I suggested having nothing, as maintaining a file is a lot of work. (Had to do tooling for pylint, but it's not ideal).